### PR TITLE
added question and answer details pages

### PR DESCRIPTION
**Pull request by mmzzyh**
see if there's anything wrong @_@

### CODE REVIEW

File: src/components/ServiceAnswerDetails.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                    value={this.state.serviceAnswer.id}
+                    onChange={(e) => this.selectServiceAnswer(e.target.value)}
+                    className="form-control">
+                    {
+                        this.state.serviceAnswers
+                            .map(serviceAnswer =>
+                                <option
+                                    value={serviceAnswer.id}
+                                    key={serviceAnswer.id}>
+                                    {serviceAnswer.choiceAnswer}
```
Comment (adamtobey) [inline src/components/ServiceAnswerDetails.js]: this should be `trueFalseAnswer`

File: src/components/ServiceQuestionDetails.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                    value={this.state.serviceQuestion.id}
+                    onChange={(e) => this.selectServiceQuestion(e.target.value)}
+                    className="form-control">
+                    {
+                        this.state.serviceQuestions
+                            .map(serviceQuestion =>
+                                <option
+                                    value={serviceQuestion.id}
+                                    key={serviceQuestion.id}>
+                                    {serviceQuestion.question}
```
Comment (adamtobey) [inline src/components/ServiceQuestionDetails.js]: this should be `serviceQuestion.title`, and check the other JSON fields against the names in our backend model

Comment (mmzzyh) [reply src/components/ServiceQuestionDetails.js]: Checked and fixed. Thanks for catching.

File: src/components/ServiceAnswerDetails.js (hunk 1 above)
Comment (mmzzyh) [reply src/components/ServiceAnswerDetails.js]: Why..?

File: src/components/ServiceQuestionDetails.js [hunk 3]
```diff
@@ -43,12 +43,12 @@ class ServiceQuestionDetails extends React.Component {
                     onChange={(e) => this.selectServiceQuestion(e.target.value)}
                     className="form-control">
                     {
-                        this.state.serviceQuestions
+                        this.state.serviceQuestios
```
Comment (adamtobey) [inline src/components/ServiceQuestionDetails.js]: typo

File: src/components/ServiceAnswers.js [hunk 4]
```diff
@@ -27,7 +27,9 @@ class ServiceAnswers extends React.Component {
                         this.state.serviceAnswers
                             .map(serviceAnswer =>
                                 <tr key={serviceAnswer.id}>
-                                    <td>{serviceAnswer.choiceAnswer}</td>
+                                    <td><Link
+                                    to={`/admin/service-answers/${serviceAnswer.id}`}>
+                                    {serviceAnswer.choiceAnswer}</td>
```
Comment (adamtobey) [inline src/components/ServiceAnswers.js]: Forgot the `</Link>` to close

File: src/components/ServiceAnswerDetails.js (hunk 1 above)
Comment (adamtobey) [reply src/components/ServiceAnswerDetails.js]: nvm, you're right

**Pull request merged**